.vscode/example/tasks.json

```diff
@@ -128,6 +128,33 @@
             "group": "build",
             "type": "shell",
             "command": "./fbt COMPACT=1 DEBUG=0 launch_app APPSRC=${relativeFileDirname}"
+        },
+        {
+            "label": "[Debug] Launch App on Flipper with Serial Console",
+            "dependsOrder": "sequence",
+            "problemMatcher": [],
+            "dependsOn": [
+                "[Debug] Launch App on Flipper",
+                "Serial Console"
+            ]
+        },
+        {
+            // Press Ctrl+] to quit
+            "label": "Serial Console",
+            "type": "shell",
+            "command": "./scripts/serial_cli.py",
+            "group": "none",
```

Comment:
for command to be visible in palette, has to be "build"

Also `            "problemMatcher": [],` is requried by [specs](https://code.visualstudio.com/docs/editor/tasks#_compound-tasks)

>Any background/watch tasks used in dependsOn with "dependsOrder": "sequence" must have a problem matcher that tracks when they are "done". The following task runs task Two, task Three, and then task One.